pom.xml

```diff
@@ -4,10 +4,9 @@
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
-        <groupId>net.wasdev.maven.parent</groupId>
-        <artifactId>java7-parent</artifactId>
-        <version>1.4</version>
-        <relativePath />
+        <groupId>net.wasdev.wlp.maven.parent</groupId>
+        <artifactId>liberty-maven-app-parent</artifactId>
+        <version>2.0-SNAPSHOT</version>
```

Comment:
For this one, since it is already using a parent, let's leave that in place and move the content of our parent pom into this pom.